content/docs/2.8/concepts/scaling-deployments.md

```diff
@@ -232,6 +232,29 @@ metadata:
 
 The presensce of this annotation will pause autoscaling no matter what number of replicas is provided. The above annotation will scale your current workload to 0 replicas and pause autoscaling. You can set the value of replicas for an object to be paused at to any arbitary number. To enable autoscaling again, simply remove the annotation from the `ScaledObject`definition.
 
+### `Activating` and `Scaling`
+
+To give a consisten solution to this problem, KEDA has 2 different phases during the autoscaling process.
+
+- **Activation phase:**: The activating (or deactivating) phase is the moment when KEDA (operator) has to decide if the workload should be scaled from/to zero. KEDA is the responsible for this action based on the result of the scaler`IsActive` function and only applies to 0<->1 scaling. There are use-cases where the activating value (0-1 and 1-0) is totally different than 0, such as workloads scaled with the Prometheus scaler where the values go from -X to X.
+- **Scaling phase:** The scaling phase is the moment when KEDA has decided to scale out to 1 instance and now is the HPA controller who takes the scaling decisions based on the configuration defined in the generated HPA (from ScaledObject data) and the metrics exposed by KEDA (metrics server). This phase applies the to 1<->N scaling.
+
+#### Managing Activation & Scaling Thresholds
+
+KEDA allows you to specify different values for each scenario:
+
+- **Activation:** Defines when the scaler is active or not and scales from/to 0 based on it.
+- **Scaling:** Defines the target value to scale the workload from 1 to _n_ instances and vice versa. To achieve this, KEDA passes the target value to the Horizontal Pod Autoscaler (HPA) and the built-in HPA controller will handle all the autoscaling.
+
+> ⚠️ **NOTE:** If the minimum replicas is >= 1, the scaler is always active and the activation value will be ignored.
+
+**Each scaler defines parameters for their use-cases, but the activation will always be the same as the scaling value, appended by the prefix `activation`** (ie: `threshold` for scaling and `activationThreshold` for activation).
```

Comment:
I don't think we need to put this in bold as it looks odd in the rendered docs IMO; but who am I!
```suggestion
Each scaler defines parameters for their use-cases, but the activation will always be the same as the scaling value, appended by the prefix `activation` (ie: `threshold` for scaling and `activationThreshold` for activation).
```

Reply:
addressed